components/Footer.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
```

Comment:
can you own this piece of code
![image](https://user-images.githubusercontent.com/47992694/108297296-11daf880-7169-11eb-9c25-235d4a29e5d2.png)